react-web/src/hooks/useLogs.ts

```diff
@@ -13,41 +14,119 @@ export const useLogs = (
     testEnded: boolean,
     handleErrorScenario: () => void
 ) => {
+    const dispatch = useDispatch()
     const [logInfo, setLogInfo] = useState<Log[]>([])
     const [fetchingLogs, setFetchingLogs] = useState(false);
     const [refetchLogsOffset] = useState(1);
+    const [enabled, setEnabled] = useState(false)
```

Comment:
see change request from below (line 33) for more details
```suggestion
```

Reply:
Done